### DISCUSSION TITLE
Asking for keyboard input for printing long text

### DISCUSSION THREAD

**Discussion by wasi-master** [Textualize/rich#1493] [Q&A] [answered]
I was thinking about something similar to how when you type `license()` while in the python interpreter (aka. repl), where it asks for you to press enter to print the next part instead of printing the whole thing at once, I was thinking if anything like that would be possible with rich? here is an image for reference ![examples](https://i.imgur.com/rtQ5vIG.png)

btw I'm using rich.markdown

**Comment by yashrathi-git** [comment, marked as answer]
Sure you could use [pager in rich](https://rich.readthedocs.io/en/latest/console.html#paging)

**Comment by willmcgugan** [comment]
You could use the builtin `input()`, or Rich [prompt](https://rich.readthedocs.io/en/latest/prompt.html)
